compiler/rustc_target/src/spec/targets/loongarch64_unknown_linux_musl.rs

```diff
@@ -0,0 +1,17 @@
+use crate::spec::{base, Target, TargetOptions};
+
+pub fn target() -> Target {
+    Target {
+        llvm_target: "loongarch64-unknown-linux-musl".into(),
+        pointer_width: 64,
+        data_layout: "e-m:e-p:64:64-i64:64-i128:128-n64-S128".into(),
+        arch: "loongarch64".into(),
+        options: TargetOptions {
+            cpu: "generic".into(),
+            features: "+f,+d".into(),
+            llvm_abiname: "lp64d".into(),
+            max_atomic_width: Some(64),
+            ..base::linux_musl::opts()
```

Comment:
Could you please set `crt_static_default = false` like some of the [other](https://github.com/rust-lang/rust/blob/b0696a5160711c068cb1f01b7437db7990d15750/compiler/rustc_target/src/spec/targets/hexagon_unknown_linux_musl.rs#L10) [musl](https://github.com/rust-lang/rust/blob/b0696a5160711c068cb1f01b7437db7990d15750/compiler/rustc_target/src/spec/targets/mips64_openwrt_linux_musl.rs#L11) [targets](https://github.com/rust-lang/rust/blob/b0696a5160711c068cb1f01b7437db7990d15750/compiler/rustc_target/src/spec/targets/mips_unknown_linux_musl.rs#L9) do? We want to eventually make [all musl targets dynamically linked by default](https://github.com/rust-lang/compiler-team/issues/422).

Reply:
Done. Thanks.